internal/boards/goto.go

```diff
@@ -23,8 +23,9 @@ func RegisterGoTo() {
 			app.Error(err.Error())
 			return
 		}
+		filter, err := api.GetFilter(boardConfig.Filter.Id)
```

Comment:
err is ignored here

Reply:
good catch! 8a3cc4023cc74ecef7c691bc8b2042fee97fd61b